src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Definition/DefinitionEndpoint.cs

```diff
@@ -59,35 +53,35 @@ public void ApplyCapabilities(VSInternalServerCapabilities serverCapabilities, V
 
     protected async override Task<DefinitionResult?> TryHandleAsync(TextDocumentPositionParams request, RazorRequestContext requestContext, DocumentPositionInfo positionInfo, CancellationToken cancellationToken)
     {
-        requestContext.Logger.LogInformation("Starting go-to-def endpoint request.");
+        Logger.LogInformation("Starting go-to-def endpoint request.");
         var documentContext = requestContext.GetRequiredDocumentContext();
 
         if (!FileKinds.IsComponent(documentContext.FileKind))
         {
-            requestContext.Logger.LogInformation("FileKind '{fileKind}' is not a component type.", documentContext.FileKind);
+            Logger.LogInformation("FileKind '{fileKind}' is not a component type.", documentContext.FileKind);
             return default;
         }
 
         // If single server support is on, then we ignore attributes, as they are better handled by delegating to Roslyn
-        var (originTagDescriptor, attributeDescriptor) = await GetOriginTagHelperBindingAsync(documentContext, positionInfo.HostDocumentIndex, SingleServerSupport, requestContext.Logger, cancellationToken).ConfigureAwait(false);
+        var (originTagDescriptor, attributeDescriptor) = await GetOriginTagHelperBindingAsync(documentContext, positionInfo.HostDocumentIndex, SingleServerSupport, Logger, cancellationToken).ConfigureAwait(false);
```

Comment:
the `GetOriginTagHelperBindingAsync` is apparently static, can it easily change to be not static so the logger won't need to be passed like this?

I might help simplify this method's usage in the test project too.

Reply:
The current tests don't construct an instance of the endpoint at all, so it would arguably make the tests slightly more complicated. In general I don't think making things non-static is a goal.

Reply:
okay thanks.